tests/spyre_util.py

```diff
@@ -580,3 +638,46 @@ def get_chicken_soup_prompts(num_prompts: int) -> list[str]:
         prompts = prompts * (math.ceil(num_prompts / 4))
 
     return prompts[:num_prompts]
+
+
+def get_longer_chicken_soup_prompts(num_prompts: int) -> list[str]:
```

Comment:
this is needed unfortunately since some of the prompt requests are pretty long in the scheduler tests

Reply:
I think there could be a refactor here with the overlap between the two prompt functions 🤔

Reply:
Opened an issue for it - https://github.com/vllm-project/vllm-spyre/issues/348